kakaemma/My-Diary-React
# #162192931 set up react project for my diary

#### What does this PR do?
This PR creates a react application for mydiary,
Adds redux to the application,
Set's up react router and also adds a landing page with a login form.

#### How should this be manually tested?
Clone the application with `git clone `
install application dependencies with `npm install`
Run the application with `npm start`
On success, a landing page with a login form should will be displayed.

#### What are the relevant pivotal tracker stories?
[#162192931](https://www.pivotaltracker.com/story/show/162192931)

#### Any background context you want to add?
N/A
